src/style.css

```diff
@@ -102,11 +103,81 @@ button {
   border: 1px solid #888;
   padding: 0.5rem 0.9rem;
   border-radius: 4px;
+  position: relative;
+}
+
+.cart-badge {
+  position: absolute;
+  top: -9px;
```

Comment:
use top 0, right 0 and translate(50%,-50%) to postiion the badge this will position  perfectly regarlesss of the size